### ISSUE TITLE
Emacs 26 display-line-numbers-mode breaks quick-peek

### ISSUE THREAD

**Issue by ermingol23** [cpitclaudel/quick-peek#14]
When display-line-numbers-mode is enabled in Emacs 26, the borders are no longer thin lines etc.; all lines within the quick-peek-popup will have the same height. Is this something that is possible to fix?

**Comment by cpitclaudel** [comment]
Good catch! Hmm. I think I need to ask on emacs-devel.  I'm not sure what the fix might be.

**Comment by cpitclaudel** [comment]
I emailed emacs-devel

**Comment by ermingol23** [comment]
One possible workaround could be to use a child frame that holds the popup.

This approach would lead to the child-frame hiding the text under it, which is a different behavior compared to how quick-peek works today. If this is not desired, one could reuse the overlay that quick-peek uses today (but with no content within it) to "push" the hidden text so it becomes visible again; the idea is that the overlay is completely hidden behind the child frame.

**Comment by ermingol23** [comment]
Any response from emacs-devel you can share?

**Comment by cpitclaudel** [comment]
https://lists.gnu.org/archive/html/emacs-devel/2019-05/msg00155.html is where the thread started.  Unfortunately, it didn't really reach a conclusion.

**Comment by mattiasdrp** [comment]
Did someone come up with a solution for this?
